src/tests/ftest/util/job_manager_utils.py

```diff
@@ -936,6 +938,10 @@ def check_logs(self, pattern, since, until, quantity=1, timeout=60):
             complete = detected == quantity
             timed_out = time.time() - start > timeout
 
+            if complete:
+                self.timestamps["running"] = datetime.now().strftime(
+                    "%Y-%m-%d %H:%M:%S")
```

Comment:
I wonder if something generic like "seconds since epoch" is better here, and let consumers decide what format they want the time in.

Reply:
The `self.timestamps` dictionary contains date/time stamps to be used with the `journalctl --since=<yyy_mm_dd hh:mm:ss>` command, hence the specific format.  The intent is for the setting of each dictionary value to made within the class functions, but the access to them is 'public' in case the user wants to call the `Systemctl.get_log_data(hosts, since, unti, timeout)` method directly in a test method.

Reply:
> The self.timestamps dictionary contains date/time stamps to be used with the journalctl --since=<yyy_mm_dd hh:mm:ss> command

I just wonder if they might get leveraged for other purposes in the future, where the format desired would be different, or a "time since" calculation might be desired, which would mean having to parse the date-string format back into a time value.

Reply:
I guess they could.  For now they are only used with `journalctl` as a means of obtaining information that used to be in the `orterun` process output.